resources/zib/ext-TreatmentDirective2.ReasonForEnding.xml

```diff
@@ -41,6 +39,10 @@
       <path value="Extension" />
       <max value="1" />
     </element>
+    <element id="Extension.extension">

```

Comment:
This seems incorrect

Reply:
I think this is an unpleasant change from the profiling tool. So it can be deleted.